### ISSUE TITLE
Discussion

### ISSUE THREAD

**Issue by lukeweston** [CCHS-Melbourne/LoliBot#1]
[This is not an "issue". It's just a comment really, and I couldn't think of a better place to put it.]

You've probably already seen these before, but just in case you haven't.

https://www.aliexpress.com/item/10pcs-lot-NodeMcu-Lua-WIFI-development-board-based-on-the-ESP8266-Internet-of-things/32339960467.html

These "WiFi smart car" kits give you two motor/gearbox/tyre assemblies plus the front dolly wheel, the laser-cut acrylic baseplate, battery holder, screws, wire, all the bits you need, a NodeMCU Amica clone (ESP8266) board, and the carrier motherboard which has the pair of H-bridges to power the motors, and supplies 5V to the MCU and connectors to wire everything neatly. So USD$14 gets you everything needed for a Wi-Fi controlled vehicle. (But only an ESP8266, and no sensors, no batteries, no software and no documentation.) I bought one and did receive what they claimed.

Maybe you can get an ESP32 board which drops in, with the same pinout and form factor?

It's a really tempting kit economically, although I do understand the fun / enjoyment value of doing DIY from scratch. Maybe you can mix and match the parts, or replace some of them yourself.

MEMS IMU devices like the MPU6050 are all over AliExpress (or the similar usual suspects) at crazy prices. (USD$1 for one breakout board is about a third of the best price I've seen for 5000x-10000x from Avnet/Digikey/Arrow etc, and the latter is the bare silicon without the breakout board.) Maybe rejects, maybe too good to be true, I don't know... caveat emptor.

https://www.aliexpress.com/item/GY-521-MPU-6050-MPU6050-Module-3-Axis-analog-gyro-sensors-3-Axis-Accelerometer-Module/1954221775.html

Anyway, have fun, hope this is valuable! :)

**Comment by nickzoic** [comment]
Yeah, that second board there is the 'motor shield' I mentioned (did I manage to find one on the night?) the other night at CCHS, it's quite handy, as well as the H bridges it has several 3-pin servo compatible headers for other I/O pins.

The 8266 is old news for some of us :-) but still quite capable of doing cool stuff in MicroPython ...
